community/import-tool/src/docs/ops/import-tool.asciidoc

```diff
@@ -97,6 +97,35 @@ For help with running the import tool using Windows PowerShell, see <<powershell
 
 include::options.adoc[]
 
+[[import-tool-output-and-statistics]]
+=== Output and statistics
+
+While an import is running through its different stages, some statistics and figures are printed in the console.
+The general interpretation of that output is to look at the horizontal line, which is divided up into sections, each section representing one type of work going on in parallel with the other sections.
+The wider a section is, the more time is spent there relative to the other sections, the widest being the bottleneck, also marked with `\*`.
+If a section has a double line, instead of just a single line, it means that multiple threads are executing the work in that section.
+To the far right a number is displayed telling how many entities (nodes or relationships) have been processed by that stage.
+
+As an example:
+
+`[*>:20,25 MB/s------------------|PREPARE(3)====================|RELATIONSHIP(2)===============] 16M`
+
+Would be interpreted as:
+
+* `>` data being read, and perhaps parsed, at `20,25 MB/s`, data that is being passed on to ...
+* `PREPARE` preparing the data for ...
+* `RELATIONSHIP` creating actual relationship records and ...
+* `v` writing the relationships to the store. This step isn't visible in this example, because it's so cheap compared to the other sections.
```

Comment:
Not sure this is the correct place to raise this issue, but why is it `v` and not `w` for write?